motor/providers/google/clients.go

```diff
@@ -11,6 +11,22 @@ import (
 	"google.golang.org/api/transport"
 )
 
+func (t *Provider) Credentials(scopes ...string) (*googleoauth.Credentials, error) {
+	ctx := context.Background()
+
+	if t.serviceAccountSubject == "" {
+		return googleoauth.FindDefaultCredentials(ctx, scopes...)
```

Comment:
i assume this uses the gcloud cli somehow? can we add a debug log here, indicating that we're falling back to cli auth? would make it easier for debugging purposes if we need that

Reply:
Great call, lets do that